waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates.hs

```diff
@@ -7,78 +7,190 @@ module Wasp.Cli.Command.CreateNewProject.StarterTemplates
 
-getStarterTemplates :: IO [StarterTemplate]
-getStarterTemplates = do
-  remoteTemplates <- fromRight [] <$> fetchRemoteStarterTemplates
-  return $ localTemplates ++ remoteTemplates ++ [AiGeneratedStarterTemplate]
-
-fetchRemoteStarterTemplates :: IO (Either String [StarterTemplate])
-fetchRemoteStarterTemplates = do
-  fmap extractTemplateNames <$> Github.fetchRemoteTemplatesGithubData
-  where
-    extractTemplateNames :: [Github.RemoteTemplateGithubData] -> [StarterTemplate]
-    -- Each folder in the repo is a template.
-    extractTemplateNames =
-      map
-        ( \metadata ->
-            RemoteStarterTemplate $
-              DirBasedTemplateMetadata
-                { _name = Github._name metadata,
-                  _path = Github._path metadata,
-                  _description = Github._description metadata
-                }
-        )
-
-localTemplates :: [StarterTemplate]
-localTemplates = [defaultStarterTemplate]
+type StartingInstructionsBuilder = String -> String
+
+{- HLINT ignore getTemplateStartingInstructions "Redundant $" -}
+
+-- | Returns instructions for running the newly created (from the template) Wasp project.
+-- Instructions assume that user is positioned right next to the just created project directory,
+-- whose name is provided via projectDirName.
+getTemplateStartingInstructions :: String -> StarterTemplate -> String
+getTemplateStartingInstructions projectDirName = \case
+  GhRepoStarterTemplate _ metadata -> _buildStartingInstructions metadata projectDirName
+  LocalStarterTemplate metadata -> _buildStartingInstructions metadata projectDirName
+  AiGeneratedStarterTemplate ->
+    unlines
+      [ styleText $ "To run your new app, do:",
+        styleCode $ "    cd " <> projectDirName,
+        styleCode $ "    wasp db migrate-dev",
+        styleCode $ "    wasp start"
+      ]
+
+getStarterTemplates :: [StarterTemplate]
+getStarterTemplates =
+  [ defaultStarterTemplate,
+    todoTsStarterTemplate,
+    openSaasStarterTemplate,
+    embeddingsStarterTemplate,
+    AiGeneratedStarterTemplate
+  ]
 
 defaultStarterTemplate :: StarterTemplate
-defaultStarterTemplate =
+defaultStarterTemplate = basicStarterTemplate
+
+{- HLINT ignore basicStarterTemplate "Redundant $" -}
+
+basicStarterTemplate :: StarterTemplate
+basicStarterTemplate =
   LocalStarterTemplate $
     DirBasedTemplateMetadata
-      { _name = "basic",
-        _path = "basic",
-        _description = "Simple starter template with a single page."
+      { _path = [reldir|basic|],
+        _name = "basic",
+        _description = "Simple starter template with a single page.",
+        _buildStartingInstructions = \projectDirName ->
+          unlines
+            [ styleText $ "To run your new app, do:",
+              styleCode $ "    cd " <> projectDirName,
+              styleCode $ "    wasp db start"
+            ]
       }
 
+{- HLINT ignore openSaasStarterTemplate "Redundant $" -}
+
+openSaasStarterTemplate :: StarterTemplate
```

Comment:
Hard coding the templates here makes it impossible to add new templates without releasing a new Wasp version. I just wanted to point that out so we are aware.

Reply:
I know, but concluded that is probably all right and not that important, and it keeps things simpler for us.